README.md

```diff
@@ -42,6 +42,22 @@ Then run the following script:
 
 `python scripts/run.py gitlab`
 
+### Github
```

Comment:
Some details what is used, and how the comments setting lanes and dates are working is needed.